### PR TITLE
[DependencyInjection] Update shared.rst to add doc on PHP attributes

### PR DESCRIPTION
**Pull request by arnissolle**
Add code block for PHP attributes.

### CODE REVIEW

File: service_container/shared.rst [hunk 1]
```diff
@@ -11,6 +11,19 @@ in your service definition:
 
 .. configuration-block::
 
+    .. code-block:: php-attributes
+
+        // src/SomeNonSharedService.php
+        namespace App\SomeNonSharedService;
```
Comment (xabbuh) [inline service_container/shared.rst]:
```suggestion
        namespace App;
```

**Review by xabbuh — CHANGES_REQUESTED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Pierre, thanks a lot for improving the docs ... and congrats on your first Symfony Docs contribution 🎉

**Comment by arnissolle** [comment]
⚠️ @javiereguiluz @xabbuh [the doc is broken](https://symfony.com/doc/current/service_container/shared.html), is there something missing in the code block?

Capture d’écran 2024-03-22 à 18 46 21

**Comment by xabbuh** [comment]
@arnissolle Thanks for the heads up. I guess this is due to the fact that there was a tab instead of four spaces (dda1b38a0acb6023236bbc5fe3d17cc15df801b1 should fix that).

**Pull request merged**